client/lib/abtest/index.js

```diff
@@ -96,7 +107,17 @@ ABTest.prototype.init = function( name ) {
 	this.defaultVariation = testConfig.defaultVariation;
 	this.variationNames = variationNames;
 	this.experimentId = name + '_' + variationDatestamp;
-	this.allowAnyLocale = testConfig.allowAnyLocale === true;
+
+	this.localeTargets = [ 'en' ];
+
+	if ( 'undefined' !== typeof testConfig.localeTargets ) {
+		if ( testConfig.localeTargets === false ) {
+			this.localeTargets = false;
+		} else {
+			this.localeTargets = testConfig.localeTargets;
+		}
+	}
```

Comment:
I'm trying to keep it simple - taking into account the current tests which all only run against "en". So no param = en, explicit false = all locales (not very likely that we'll ever do this imo), and array = one or more specific values.